src/iocore/net/SSLUtils.cc

```diff
@@ -1086,6 +1086,20 @@ ssl_callback_info(const SSL *ssl, int where, int ret)
       }
       Metrics::Counter::increment(it->second);
     }
+
+#if defined(OPENSSL_IS_BORINGSSL) || defined(SSL_get_negotiated_group)
+    // TLS Group
+#ifdef OPENSSL_IS_BORINGSSL
+    int nid = SSL_get_negotiated_group(ssl);
+#elif defined(SSL_get_negotiated_group)
+    int nid = SSL_get_negotiated_group(const_cast<SSL *>(ssl));
+#endif
+    if (nid != NID_undef) {
+      if (auto it = tls_group_map.find(nid); it != tls_group_map.end()) {
```

Comment:
We should have OTHER, otherwise we'd see a mysterious drop when SSL libraries and clients support a new group.
https://github.com/apache/trafficserver/pull/9623

And the table could be autogenerated if the library used has `SSL_get_all_group_names`. Only users who use SSL library without the function would be affected if new groups are added.

Reply:
Yeh, I was thinking the same thing. Will do them.

Reply:
BoringSSL has an API to get NID from group name?

Reply:
No, it doesn't. Look at `cipher_map`. Its key is a string, not NID. It may not be great in terms of calculating the hash, but if we really care about it, I suppose we could use the pointer for the const char.